src/main/java/pl/pjwstk/woloappapi/utils/EmailUtil.java

```diff
@@ -19,7 +19,7 @@ public class EmailUtil {
     public void sendResetPasswordEmail(String email) throws MessagingException {
         String message = String.format("""
         <div>
-          <a href="http://localhost:8080/reset-password?email=%s" target="_blank">click link to set password</a>
+          <a href="http://localhost:3000/reset-password?email=%s" target="_blank">click link to set password</a>
```

Comment:
może warto stworzyć w konfigu zmienną sklejącą 3 zmienne:

```
$app_full_domain = $protocol + $domain + $port
```

dzięki temu można tych wszystkich zmiennych używać wszędzie, a w razie zmiany podmienia się 1 miejsce. Nawet jeśli teraz używacie to tylko w 1 miejscu, to zaraz ktoś na pewno gdzieś to skopiuje :)